scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -77,22 +79,26 @@ def export_standalone_project(ctx: exp.O3DEScriptExportContext,
     """
 
     is_installer_sdk = manifest.is_sdk_engine(engine_path=ctx.engine_path)
-
+    
     # Use a provided logger or get the current system one
     if not logger:
         logger = logging.getLogger()
         logger.setLevel(logging.ERROR)
 
     # Calculate the tools and game build paths
     default_base_build_path = ctx.engine_path / 'build' if engine_centric else ctx.project_path / 'build'
-    if not tools_build_path:
-        if is_installer_sdk:
+    
+    if is_installer_sdk:
```

Comment:
Possible to throw an error message and stop the script early if is_installer_sdk and monolithic_build?

Reply:
I think this is related to @lumberyard-employee-dm's suggestion to check for monolithic artifacts in the event of monolithic build, and if not detected then give an error. We plan on having monolithic available in the event of a release build, which is handled separately from config for profile and release.

I think for the above reasons I'm holding off atm.